src/curriculum/dataflow/dataflow.json

```diff
@@ -0,0 +1,39 @@
+{
+  "title": "Dataflow",
```

Comment:
Just as a note, I've always preferred writing Dataflow as you did here: "Dataflow".  It's sometimes written different ways ("DataFlow", "Data Flow", etc.), but I like standardizing on this (no space, no capital F).

Reply:
I went to the project page and read the recently posted blog post to see how it was done in "official" documents.

Reply:
Sherry and Lisa at one point both agreed on "Dataflow" when I asked in a meeting (since I was adding the text to a topbar in the UI) so I agree that this is the right way to write it out.